UKHomeOffice/cerberus-service
# Cop 10222 air pax task list

## Description
This PR adds the creation of the the task card for a target Task.

**Note**
- The UI view is not connected to the active API, it will always run the 'catch' condition
- The catch condition currently sets the activity log data
- The unit tests mock connecting to the API


**Note**
This PR is also made up of the following approved PRs
- https://github.com/UKHomeOffice/cerberus-service/pull/537
- https://github.com/UKHomeOffice/cerberus-service/pull/541

## To Test
- Pull and run locally
- Copy the JSON from `taskData_AirPax_NoAssignee.fixture.json` or `taskData_AirPax_AssigneeOtherUser.fixture.json` or `taskData_AirPax_AssigneeCurrentUser.fixture.json` file and create an object in `TaskListPage` by pasting the data within `const tempData = {...}`
- Go to `http://localhost:8080/airpax/tasks`
- The task list page should display a `TaskListPage` with a task that resembles the screenshot below:
<img width="984" alt="image" src="https://user-images.githubusercontent.com/19333750/164445497-64607601-0fa7-4563-a471-0305e77ed5ca.png">


## Developer Checklist

\* Required
